.github/ISSUE_TEMPLATE/questions-or-help.md

```diff
@@ -1,13 +1,15 @@
 ---
 name: Questions or Help
-about: For general questions, please use our Slack channel or mailing list.
+about: Do NOT post usage questions here. Please use Slack or our mailing list instead.
```

Comment:
Should there be a link to Slack invitation, and possibly mention stackoverflow?  Mailing list is great, but it is not very well searched by google, and tend not to show up in query results, making people ask the same questions.

Reply:
P.S. Assuming links will work in the `about` section.  If not, the links below are good too.